preprod.gen3.biodatacatalyst.nhlbi.nih.gov/manifest.json

```diff
@@ -7,29 +7,29 @@
     "autodeploy": "yes"
   },
   "versions": {
-    "access-backend": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/access-backend:1.3",
-    "arborist": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/arborist:2020.11",
+    "access-backend": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/access-backend:2020.12",
```

Comment:
@themarcelor we either need to add this to our monthly releases (my vote) or don't try to use the monthly release tag here. I don't see the tag in this repo

Reply:
Yeah. It hasn't been added yet. We'l work on it in Q1 2021.